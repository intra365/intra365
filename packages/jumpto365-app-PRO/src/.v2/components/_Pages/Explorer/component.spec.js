import React, { Component } from 'react';

import ExplorerPage from '.'

test('Check that it renders', () => {

    const wrapper = shallow(<ExplorerPage context={mocks.user()}/>);

    expect(wrapper).toMatchSnapshot();
 
}

)