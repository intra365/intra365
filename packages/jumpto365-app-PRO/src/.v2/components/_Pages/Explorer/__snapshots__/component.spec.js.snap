// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check that it renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <UserPage
    context={Object {}}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): "Not authenticated",
  Symbol(enzyme.__nodes__): Array [
    "Not authenticated",
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
